components/next_steps_view/next_steps_view.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import professionalLogo from 'images/cloud-logos/professional.svg';
+
+import './next_steps_view.scss';
+
+type Props = {
+    skuName: string;
+};
+
+export default class NextStepsView extends React.PureComponent<Props> {
+    getBottomText = () => {
+        // lmao
+        const {isFinished} = {isFinished: false};
+
+        if (isFinished) {
```

Comment:
Is this just setting up for the future?

Reply:
Yeah, there should be a user preference storing this normally so we'll pull that in when the time comes.